.idea/codeStyles/Project.xml

```diff
@@ -0,0 +1,10 @@
+<component name="ProjectCodeStyleConfiguration">
```

Comment:
@miloshavlicek Pls remove PhpStorm configuration.

Reply:
It's good practice to have code style settings in repository, because it's a pain otherwise.

Reply:
Do you agree with me or do you have another opinion for the future?

Reply:
We will leave it in this repository as you suggest. Personally, I prefer code style not to commit, because I use a global settings.

Reply:
@Sitole What do you think about?

Reply:
I'm not sure.
I think `.editorconfig` is more more important and `.idea` setting is messy.
I´m using Intellij instead of PHPStorm and i´m not sure if `.idea` files are compatible.

@janbarasek I prefer global standards (maybe we should use this cool thing https://www.jetbrains.com/help/phpstorm/settings-tools-settings-repository.html)

Reply:
Language-specific code style settings are compatible across Jetbrains IDEs. Unfortunately `.editorconfig` is not able to solve this problem. The goal is zero setup for new contributtors. 
In other languages:
In Node.js repositories I use https://prettier.io/ and Webstorm implements it natively.
C# has its standards and project settings are committed to repositories.

Reply:
Sounds interesting, but can you show some popular projects using .idea files?

Reply:
I was browsing popular packages and decided to delete the .idea directory.